src/style.css

```
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+.search {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

```suggestion
  font-family: inherit;
```
Too make it work apply font-family with correct values to the `html` tag selector

---

done